Packs/CybleEventsV2/Integrations/CybleEventsV2/README.md

```diff
@@ -13,7 +13,9 @@ This integration was integrated and tested with version 2.0 of cybleeventsv2
     | Access Token | Access Token | True |
     | Trust any certificate (not secure) |  | False |
     | Use system proxy settings |  | False |
-    | Incident Fetch Limit | Maximum incidents to be fetched every time. Upper limit is 50 incidents. | False |
+    | Incident Fetch Limit | Maximum incidents to be fetched every time. Upper limit is 50 incidents | False |
+    | Hide (CVV/Expiry) | Check the box if you wish to hide CVV and Expiry date of card | False | 
```

Comment:
I think it's better not to use personal pronouns ("you", "check the box").
can you think of something else?

Reply:
Done